lnrpc/routerrpc/router_backend.go

```diff
@@ -967,6 +967,9 @@ func (r *RouterBackend) extractIntentFromSendRequest(
 			// pseudo-reusable, e.g. the invoice parameters are
 			// reused (amt, cltv, hop hints, etc) even though the
 			// payments will share different payment hashes.
+			//
```

Comment:
I don't really get this usecase tho, because for AMP payments we don't have to change the payment address because the setID will give us the possibility to have reusable AMP invoices, and this will only work if the peer has spontaneous payments enabled. Should we probably get rid of it @bhandras ?